document data export (#2756)
giantswarm/docs: src/content/overview/observability/data-management/data-import-export/index.md

## Why importing and exporting data

Data import and export capabilities open up powerful integration possibilities:

**Data Import Benefits:**

- **External data sources**: Send logs and events from SaaS applications, databases, or other infrastructure not managed by Giant Swarm
- **Cross-platform correlation**: Combine data from multiple environments and platforms in a single observability stack
- **Legacy system integration**: Import data from existing monitoring tools during migrations or for hybrid deployments
- **Third-party services**: Collect observability data from external services, APIs, or cloud providers

**Data Export Benefits:**

- **External monitoring tools**: Connect your existing Grafana instances, monitoring dashboards, or business intelligence tools
- **Specialized analysis**: Use advanced analytics tools, machine learning platforms, or custom applications with your observability data
- **Backup and archival**: Create additional copies of your observability data for compliance or long-term analysis
- **Multi-cloud strategies**: Centralize observability data from multiple cloud providers and platforms

## How data import and export works

> The Observability Platform API provides both **data ingestion** (sending data to the platform) and **data export** (retrieving data from the platform) capabilities through a unified, secure interface.

Reviewer: ```suggestion
The Observability Platform API provides both **data import** (sending data to the platform) and **data export** (retrieving data from the platform) capabilities through a unified, secure interface.
```

### Architecture overview

The API consists of different ingress components that use:

- **Shared host**: Based on your Giant Swarm installation's base domain (`https://observability.<domain_name>`)
- **OIDC authentication**: Secure access through your identity provider
- **Multi-tenant access control**: Tenant-scoped data access through HTTP headers

![Data export architecture](./observability-platform-api-graph.png)
[_Full size architecture diagram_](./observability-platform-api-graph-big.png)

### Authentication and access control

All data import and export requests require:

1. **Valid OIDC token**: Authentication through your organization's identity provider
2. **Tenant specification**: Include an `X-Scope-OrgId` HTTP header with an existing tenant name

Please note that your identity must have access to the specified tenant.

> ⚠️ **Important**: Only data from tenants defined in [Grafana Organization]({{< relref "/overview/observability/configuration/multi-tenancy/" >}}) resources can be accessed. Requests for non-existent tenants will be rejected.

Reviewer: ```suggestion
⚠️ **Important**: Only tenants defined in [Grafana Organization]({{< relref "/overview/observability/configuration/multi-tenancy/" >}}) resources can be used to import or export data. Requests for non-existent tenants will be rejected. 
```
